src/test/resources/concordion/mockserver/SetResponseHttpStatusCode.html

```diff
@@ -0,0 +1,39 @@
+<html xmlns:c="http://www.concordion.org/2007/concordion"
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Insert a <!DOCTYPE> declaration to before this <html> tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ADoctypePresenceCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a <title> tag to this page. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutTitleCheck)